examples/keras_io/tensorflow/generative/cyclegan.py

```diff
@@ -0,0 +1,667 @@
+"""
+Title: CycleGAN
+Author: [A_K_Nain](https://twitter.com/A_K_Nain)
+Date created: 2020/08/12
+Last modified: 2020/08/12
+Description: Implementation of CycleGAN.
+Accelerator: GPU
+"""
+
+"""
+## CycleGAN
+
+CycleGAN is a model that aims to solve the image-to-image translation
+problem. The goal of the image-to-image translation problem is to learn the
+mapping between an input image and an output image using a training set of
+aligned image pairs. However, obtaining paired examples isn't always feasible.
+CycleGAN tries to learn this mapping without requiring paired input-output images,
+using cycle-consistent adversarial networks.
+
+- [Paper](https://arxiv.org/pdf/1703.10593.pdf)
+- [Original implementation](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix)
+"""
+
+"""
+## Setup
+"""
+
+#!pip install -q git+https://github.com/keras-team/keras-core
+
+import numpy as np
+import matplotlib.pyplot as plt
+import keras_core as keras
+from keras_core import layers
+from keras_core import ops
+
+import tensorflow # tf.GradientTap for training
+import tensorflow.data as tf_data
+import tensorflow.image as tf_image
```

Comment:
Likewise you can just use `tf.data` and `tf.image` directly

Reply:
done